src/analyses/loopTermination.ml

```diff
@@ -74,7 +74,7 @@ struct
     | Queries.MustTermAllLoops ->
       let multithreaded = ctx.ask Queries.IsEverMultiThreaded in
       if multithreaded then (
-        M.warn ~category:Termination "The program might not terminate! (Multithreaded)\n";
+        M.warn ~category:Termination "The program might not terminate! (Multithreaded)";
```

Comment:
Does this PR  also make changes to warnings?

Reply:
Only the handful of instances that accidentally end with `\n` and thus introduce random empty lines between warnings. These are typos from having had to keep in mind the two different requirements for newlines for messages/logs and tracing. By normalizing all three, none of them should end with a `\n`.